src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -566,7 +566,7 @@ public override Task<bool> InvokeAsync()
             {
                 if (ticket.Principal != null)
                 {
-                    Request.HttpContext.Response.SignIn(ticket.Properties, ticket.Principal.Identities);
+                    Request.HttpContext.Response.SignIn(ticket.AuthenticationScheme, ticket.Principal, ticket.Properties);
```

Comment:
I opened a ticket for that (https://github.com/aspnet/Security/issues/167) and submitted a PR (https://github.com/aspnet/Security/pull/168) that fixes it :smile: